### ISSUE TITLE
How to GIT

### ISSUE THREAD

**Issue by CosticaPuntaru** [EveryMatrix/workshops#21]
tips and tricks and guidelines on how to use github, 

i know that github is not the repo used company wide, but having a bug in a 3rd party component may require adding issues to that component, 
knowing how to write a proper issue can help
knolage on how to use "watch" "star" "fork", submitting a pr can be useful

**Comment by CosticaPuntaru** [comment]
@arcade  or @fk1blow  do you offer your knowledge?, i'm not a good github user and from what i know you have some experience with it

**Comment by iliutastoica** [comment]
https://www.codecademy.com/learn/learn-git

:+1:

**Comment by CosticaPuntaru** [comment]
no, i don't want to make it about git, how to use git cli and things
i want to do it about how an issue needs to be written, how a pr should look, the description, title, labels, and maybe other things i don't know about

**Comment by CosticaPuntaru** [comment]
for example: ctrl + enter in the comment section posts the comment, not needing to use the mouse to press comment button is a nice feature to know that it exists

**Comment by fk1blow** [comment]
well, one source would be the `CONTRIBUTING` doc that every major/important repo has.
As an example, this is the doc for the reactjs https://github.com/facebook/react/blob/master/CONTRIBUTING.md

**Comment by CosticaPuntaru** [comment]
is there something similar for tips and tricks, like i've exemplified above ?

**Comment by CosticaPuntaru** [comment]
knowing markdown, with the github flavor can help a lot: https://guides.github.com/features/mastering-markdown/

**Comment by fk1blow** [comment]
dis - https://github.com/git-tips/tips

**Comment by fk1blow** [comment]
much better - https://github.com/tiimgreen/github-cheat-sheet

**Comment by raulvasile** [comment]
+1

Also, change the name of the issue from `How to GitHub` into `How to Git` ... the platform (GitHub, GitLab, BitBucket, etc.) is irrelevant.

**Comment by CosticaPuntaru** [comment]
ctrl + enter is not supported by gitlab

**Comment by Clokze** [comment]
Actually I suggest we don't learn platform specific shortcuts except the **?** (question mark) command that works for both gitlab and github and which will present a contextual help menu with supported key bindings for the page you are viewing

**Comment by fk1blow** [comment]
i agree with @arcade on this one! 
There is no reason to "learn" interfaces that work with git - sourcetree, github desktop, github web, gitlab, etc.
I believe there's a more interesting approach to learn git as an app, conventions, best practices, etc
